javascript/README.md

```diff
@@ -18,3 +18,10 @@ Running
 ```
 npm start
 ```
+
+How the Game is Played
```

Comment:
Started documenting rules to help refactor further (this was one of the last changes).